src/registerListeners.ts

```diff
@@ -20,5 +22,8 @@ function onDidSaveTextDocument(doc: TextDocument): void {
     }
 
     lastUploadTime = Date.now();
-    ext.reporter.sendTelemetryEvent('dockerfilesave', { "lineCount": doc.lineCount.toString() }, {});
+    ext.reporter.sendTelemetryErrorEvent('dockerfilesave', { "lineCount": doc.lineCount.toString() }, {});

```

Comment:
why line count is considered errorEvent? There is nothing to mask like filepath or user name.

Reply:
It's not an error, but as part of this ongoing mess with telemetry and `vscode-extension-telemetry`, `ITelemetryReporter` removed `sendTelemetryEvent`, keeping only `sendTelemetryErrorEvent`. I think that will only be temporary, looks like that interface is going to be removed entirely.

Reply:
I am not sure what we will do for `TelemetryReporterProxy`...

Reply:
Here's the thread for more info on `ITelemetryReporter` maybe leaving: https://github.com/microsoft/vscode-azuretools/issues/693